scripts/gen_travis.py

```diff
@@ -125,7 +134,9 @@ def __eq__(self, obj):
     '--disable-libdl',
     '--enable-opt-safety-checks',
     '--with-lg-page=16',
+    "--with-lg-page=16 --with-lg-hugepage-29",
```

Comment:
All other options have single quotes, but last one have double quotes.

Reply:
Also, I am not an expert in Travis config to be honest, so this might be wrong. My understanding is that we generate combinations from `configure_flag_unusuals` to generate and test all unusual configurations of combinations. If this is true, probably we can omit `--with-lg-page=16` part and add only `--with-lg-hugepage=29` here, so combination `--with-lg-page=16 --with-lg-hugepage=29` will be generated as well.

Reply:
>  add only --with-lg-hugepage=29 here, so combination --with-lg-page=16 --with-lg-hugepage=29 will be generated as well

For arm, not really. The number of unusual setting combined together is controlled by `max_unusual_opts` [here](https://github.com/jemalloc/jemalloc/blob/dev/scripts/gen_travis.py#L243) and it is set to 1 for arm (because we don't want that many arm tests for now). Still this raises an interesting question, should we just test with 512MB hugepage or should we couple it with 64KB pages. While in jemalloc, the two should be independent and that's why we have 64KB page tests alone before, I don't have a preference here. Please suggest.

Also FYI, I intend to put the travis change configuration in another PR because I think it may need some more discussions like this. Let's redirect the discussion [there](https://github.com/jemalloc/jemalloc/pull/2771). I am adding that commit in this PR mainly for testing purposes so that we know the test is fixed properly.